indy/kms.go

```diff
@@ -40,7 +40,7 @@ func (k *KMS) Add(KID, verKey string) {
 	k.kms.keyIDByVerKey[verKey] = KID
 }
 
-func (k *KMS) Create(_ kms.KeyType) (string, interface{}, error) {
+func (k *KMS) Create(_ kms.KeyType, _ ...kms.KeyOpts) (string, interface{}, error) {
```

Comment:
FYI, I tried to google if there is any way to annotate functions 'out of coverage check' but didn't find it for Go. C++ had those... Just started to think about these interface implementers and how 'false positive' these coverage warnings are 🤔